packages/fccanalyses/package.py

```diff
@@ -44,11 +45,13 @@ class Fccanalyses(CMakePackage, Key4hepPackage):
     depends_on('py-awkward@1.4.0', when='@:0.6.0')
     depends_on('fcc-edm', when="@:0.2.9")
     depends_on('acts@5.00.0', when="@0.3.0:0.3.4 +acts")
-    depends_on('acts@6.00.0:', when='@0.3.5: +acts')
+    depends_on('acts@6.00.0:19.5.0', when='@0.3.5:0.6.0 +acts')
+    depends_on('acts@19.6.0:', when='@0.7.0: +acts')
     depends_on('eigen', when="@0.3.0:")
     depends_on('dd4hep', when="@0.3.3: +dd4hep")
     depends_on('py-pyyaml', type=('build', 'run'))
     depends_on('py-onnx-runtime', when='+onnx')
+    depends_on('delphes@3.5.1pre07', when='@0.7.0')
```

Comment:
Does this dependency happen only for 0.7.0? I don't like that whenever there is a next version that presumably will depend on delphes someone has to add it here or it won't build since delphes won't be a dependency. Maybe for 0.7.0 onward as I put in the suggestion below?
```suggestion
    depends_on('delphes@3.5.1pre07', when='@0.7.0:')
```

Reply:
Yes, I meant to make Delphes dependency from version 0.7.0.